packages/core/src/config/predefined.ts

```diff
@@ -169,6 +169,63 @@ const TESTNET_SPORE_CONFIG: SporeConfig<PredefinedSporeConfigScriptName> = {
   },
 };
 
+export type PredefinedMainnetSporeScriptName = 'Spore' | 'Cluster';
+
+const MAINNET_SPORE_CONFIG: SporeConfig<PredefinedMainnetSporeScriptName> = {
+  lumos: predefined.LINA,
+  ckbNodeUrl: 'https://mainnet.ckb.dev/rpc',
+  ckbIndexerUrl: 'https://mainnet.ckb.dev/indexer',
+  maxTransactionSize: 500 * 1024, // 500 KB
+  defaultTags: ['latest'],
+  scripts: {
+    Spore: {
+      versions: [
+        {
+          tags: ['v2', 'latest'],
+          script: {
+            codeHash: '0x9cb02fa8621d79fff300b910d52665f594d3ad468142a66213cfaaca0409502c',
+            hashType: 'data1',
+          },
+          cellDep: {
+            outPoint: {
+              txHash: '0xcb67e11a39594a183eae50664073a85fa0140ae25480df61114aca0e47726d38',
+              index: '0x0',
+            },
+            depType: 'code',
+          },
+          behaviors: {
+            lockProxy: true,
+            cobuild: true,
+          },
+        },
+      ],
+    },
+    Cluster: {
+      versions: [
+        {
+          tags: ['v2', 'latest'],
+          script: {
+            codeHash: '0x7366a61534fa7c7e6225ecc0d828ea3b5366adec2b58206f2ee84995fe030075',
+            hashType: 'data1',
+          },
+          cellDep: {
+            outPoint: {
+              txHash: '0xe464b7fb9311c5e2820e61c99afc615d6b98bdefbe318c34868c010cbd0dc938',
+              index: '0x0',
+            },
+            depType: 'code',
+          },
+          behaviors: {
+            lockProxy: true,
+            cobuild: true,
+          },
+        },
+      ],
+    },
+  },
+};
+
 export const predefinedSporeConfigs = {
   Aggron4: TESTNET_SPORE_CONFIG,
+  Lina: MAINNET_SPORE_CONFIG,
```

Comment:
I know, Lumos is using Aggron4 and Lina.
But Aggron4 and Lina seem to be old names.

See also https://docs.nervos.org/docs/basics/tools/#ckb-explorer
[Mirana Mainnet](https://explorer.nervos.org/) | [Pudge Testnet](https://pudge.explorer.nervos.org/)

Do you think the following names are clearer?
Aggron4 -> testnet
Lina -> mainnet


anyway, ^_^
This is just a naming discussion, not request for change.

Reply:
<img width="763" alt="image" src="https://github.com/sporeprotocol/spore-sdk/assets/1297478/dc22afd2-732e-4f74-a929-110d2fb753ea">
https://github.com/nervosnetwork/ckb/blob/develop/README.md#join-a-network

Reply:
You're right, maybe don't have to following the naming of lumos. I'll mark the Aggron4 as deprecated.

Reply:
Marked `Aggron4` as deprecated, using `Testnet` and `Mainnet` instead.

Reply:
> Marked `Aggron4` as deprecated, using `Testnet` and `Mainnet` instead.

https://github.com/sporeprotocol/spore-sdk/blob/8810349eae713f1fad5c28ae235c31dde9cd3b91/examples/secp256k1/utils/config.ts#L9

Do need to modify this part in the example?

Reply:
> Do need to modify this part in the example?

Good note, I think I can update it in another PR.